tests/integration/lib/instance.rb

```diff
@@ -38,6 +38,15 @@ def run_konnector(slug, account_id)
     @stack.run_konnector self, slug, account_id
   end
 
+  def run_job(type, args)
+    @stack.run_job self, type, args
+  end
+
+  def db_prefix()
```

Comment:
It looks like the `db_prefix` method is not used. I'd prefer to not add methods that are not used (dead code).